backend/coins/btc/account.go

```diff
@@ -275,7 +275,7 @@ func (account *Account) Initialize() error {
 	}
 	account.notifier = account.Config().GetNotifier(signingConfigurations)
 
-	accountIdentifier := fmt.Sprintf("account-%s-%s", signingConfigurations.Hash(), account.Config().Code)
+	accountIdentifier := fmt.Sprintf("account-%s", account.Config().Code)
```

Comment:
wait a second... what is account.Config().Code?

Reply:
> I mean, I see `account.Code` in `emitAccountsStatusChanged` and here's `account.Config().Code`. Is it the same?

I see no `account.Code` in `emitAccountsStatusChanged`.

`account.Config().Code` is the account code. It can't contain the xpub as it could be multiple xpubs, etc. 

:-1: to renaming account code to account identifier now. Not because i don't like the rename (I love it!), but because there's a million places to update and I don't want to do that now.

Reply:
> Is that erc20TokenCode unique? I

No, but that is on our TODO list, and you said you were gonna make a PR to make it unique :) LMK if I should make the PR.

>and btw, I find it quite strange that the snippet commented as "load ERC20 tokens" passes nil in the createAndAddAccount's arg named activeTokens.

ETH has tokens. A token has no tokens. I guess a comment would help.

Reply:
> No, but that is on our TODO list

That's what I thought. Just checking. :)
I'll try to make the change. Maybe it'll help me understand better how to restructure in the future. We'll see.